src/views/AddBlog.vue

```diff
@@ -52,10 +50,10 @@ export default {
   methods: {
     AddBlog() {
       // Add a new document in collection "blogs"
-      const db = app.firestore();
```

Comment:
...here.
Wait, Why did you remove that? "Firebase" can be considered as a better representation but currently this change is not required.